servers/rendering/rendering_device.cpp

```diff
@@ -6106,8 +6583,10 @@ Error RenderingDevice::initialize(RenderingContextDriver *p_context, DisplayServ
 		String rendering_method;
 		if (OS::get_singleton()->get_current_rendering_method() == "mobile") {
 			rendering_method = "Forward Mobile";
-		} else {
+		} else if (OS::get_singleton()->get_current_rendering_method() == "forward_plus") {
 			rendering_method = "Forward+";
+		} else {
+			rendering_method = "Raytracing";
```

Comment:
Remember to undo this change from this commit, as the PR shouldn't introduce a new rendering method as per @dariosamo's comment above.